react/features/feedback/functions.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import jwtDecode from 'jwt-decode';
+
+import logger from './logger';
+
+/**
+ * Sends feedback metadata to JaaS endpoints.
+ *
+ * @param {string} url - The JaaS metadata endpoint URL.
+ * @param {Object} conference - The conference.
+ * @param {Object} feedbackData - The feedback data object.
+ * @returns {Promise}
+ */
+export async function sendFeedbackToJaaSRequest(url: string, conference: Object, feedbackData: Object) {
+    const { token } = conference.xmpp;
```

Comment:
conference.xmpp  is not considered of the public API of lib-jitsi-meet. Anyway isn't the token and all the payload information already stored  in redux?

Reply:
changed to using the value from store